backend/src/main/java/capstone/storage/backend/item/models/ItemResponse.java

```diff
@@ -0,0 +1,4 @@
+package capstone.storage.backend.item.models;
+
+public record ItemResponse(String name, String ean, String categoryName, String issuingCountry) {
```

Comment:
Ich schlage vor den Klassennamen zu überdenken. Aus Sicht der externen API natürlich eine `Response` aber aus unserer Sicht vielleicht auch ein Objekt, dass wir von einer Methode in die andere als `Parameter` übergeben wollen, etc.
Lieber entsprechend der fachlichen Bedeutung benennen.

Vorschlag:
```suggestion
public record Product(String name, String ean, String categoryName, String issuingCountry) {
```